domain-mapping/cli/domain.php

```diff
@@ -0,0 +1,352 @@
+<?php
+
+defined( 'ABSPATH' ) || die;
+
+class DarkMatter_Domain_CLI {
+    /**
+     * Add a domain to a site on the WordPress Network.
```

Comment:
The command has no descripting when running `wp`. Haven't figured out how to solve that myself either.
I see ElasticPress makes it work as follows:
```
WP_CLI::add_command( 'elasticpress', 'ElasticPress_CLI_Command' );

/**
 * CLI Commands for ElasticPress
 *
 */
class ElasticPress_CLI_Command extends WP_CLI_Command {
```
![image](https://user-images.githubusercontent.com/594871/58263493-ffeddb00-7d73-11e9-829f-d9a520e0b7cf.png)